app/tests/workstations_tests/test_session_control.py

```diff
@@ -0,0 +1,85 @@
+import json
+from urllib.parse import urlparse
+
+import pytest
+from django.conf import settings
+from django.urls import reverse
+from django.views.generic import TemplateView
+
+
+class SessionControlView(TemplateView):
+    template_name = "session_control.html"
+
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data()
+        context.update(
+            {
+                "netloc": settings.DJANGO_LIVE_TEST_SERVER_ADDRESS,
+                "domain": settings.DJANGO_LIVE_TEST_SERVER_ADDRESS.split(":")[
+                    0
+                ],
+            }
+        )
+        return context
+
+
+class WorkstationView(TemplateView):
+    template_name = "workstation.html"
+
+
+class SessionCreationView(TemplateView):
+    template_name = "new_session.html"
+
+
+@pytest.mark.playwright
+def test_viewer_session_control(live_server, page, settings):
+    url = urlparse(live_server.url)
+    settings.DJANGO_LIVE_TEST_SERVER_ADDRESS = f"{url.netloc}"
```

Comment:
Does this variable still need to be set? If not, is there a need to parse `live_server.url`? Can't that just be used directly?

Reply:
It still needs to be set because we need it in the view template. It doesn't serve any other purpose than that, so we could call it anything.

Reply:
Really, the values don't appear to be used in the templates? Or am I missing something?

Reply:
Ah wait, domain is, netloc isn't, but maybe neither need to be handled if the other comment is solved to do more things on the Python side.

Reply:
Indeed no longer necessary after the rework from the above comment!